portal-backend/depmap/data_access/interface.py

```diff
@@ -399,36 +410,13 @@ def get_custom_cell_lines_dataset() -> str:
     return interactive_utils.get_custom_cell_lines_dataset()
 
 
-def get_matrix_id(dataset_id: str) -> int:
-    """
-    Load the matrix id for the given dataset.
-    Matrices are specific to the legacy data access implementation
-    """
-    return interactive_utils.get_matrix_id(dataset_id)
-
-
 def has_config(dataset_id: str) -> bool:
     """
     Check whether the given dataset exists in interactive config
     """
     return interactive_utils.has_config(dataset_id)
 
 
-def is_filter(dataset_id: str) -> bool:
-    """
-    Check whether the given dataset is a context or custom cell lines dataset.
-    """
-    return interactive_utils.is_filter(dataset_id)
-
-
-def is_standard(dataset_id: str) -> bool:
-    """
-    Check whether the given dataset is standard or nonstandard. 
-    Only applicable for the legacy data access implementation.
-    """
-    return interactive_utils.is_standard(dataset_id)
-
-
```

Comment:
Unrelated: a bunch of these deprecated functions aren't being used anymore 🎉 (at least in the modules that reference the `data_access` interface) so I'm removing them from the interface.